Mathlib/LinearAlgebra/Matrix/ProjectiveSpecialLinearGroup.lean

```diff
@@ -0,0 +1,33 @@
+/-
+Copyright (c) 2023 Wen Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Wen Yang
+-/
+import Mathlib.LinearAlgebra.Matrix.SpecialLinearGroup
+
+/-!
+# Projective Special Linear Group
+
+## Notation
+
+In the `MatrixGroups` locale:
+
+* `PSL(n, R)` is a shorthand for `Matrix.ProjectiveSpecialLinearGroup (Fin n) R`
+-/
+
+namespace Matrix
+
+universe u v
+
+open Matrix LinearMap
+
+open scoped MatrixGroups
+
+variable (n : Type u) [DecidableEq n] [Fintype n] (R : Type v) [CommRing R]
+
+/-- A projective special linear group is the quotient of a special linear group by its center.-/
+abbrev ProjectiveSpecialLinearGroup : Type _ :=
+    SpecialLinearGroup n R ⧸ Subgroup.center (SpecialLinearGroup n R)
```

Comment:
Is `X / Subgroup.Center X` something worth naming in its own right?

Reply:
I think there is no standard name for this, so it would only confuse people.